.eslintrc.json

```
@@ -9,6 +347,8 @@
 				"SwitchCase": 1
 			}
 		],
-		"jsdoc/newline-after-description": ["error", "never"]
+		"jsdoc/newline-after-description": ["error", "never"],
+		"@typescript-eslint/ban-ts-comment": "off",
+		"@typescript-eslint/consistent-type-imports": "off"
```

```suggestion
		"@typescript-eslint/consistent-type-imports": [
			"error",
			{
				"prefer": "type-imports",
				"disallowTypeAnnotations": true
			}
		]
```

I don't want either of these off